build-scripts/x64-windows-static.cmake

```diff
@@ -0,0 +1,5 @@
+set(VCPKG_TARGET_ARCHITECTURE x64)
```

Comment:
I'm confused why we're basically copying https://github.com/CleverRaven/Cataclysm-DDA/blob/master/.github/vcpkg_triplets/x64-windows-static.cmake here. Can we reuse them somehow?

Reply:
I followed your suggestion in https://github.com/CleverRaven/Cataclysm-DDA/pull/56875#discussion_r867399662